docs/sources/mimir/configure/configure-tsdb-block-upload.md

```diff
@@ -55,10 +55,12 @@ We intend to add validation of uploaded blocks in a future release, which would
 
 ### The results-cache needs flushing
 
-After uploading one or more blocks, the results-cache needs flushing. The reason is that Grafana Mimir caches query results
-for queries that don’t touch the most recent 10 minutes of data. After uploading blocks however, queries may return different
-results (because new data was uploaded). Therefore cached results may be wrong, meaning the cache should manually be flushed
-after uploading blocks.
+Grafana Mimir caches query results for queries that don’t require the most recent 10 minutes of data. 
+After uploading blocks however, queries may return different results – because new data was uploaded.
+This means that cached results may be wrong.
+To fix the cache results, Mimir operator can manually flush the results cache.
+Possible alternative is to decrease time-to-live period for cache results from default 7 days to shorter period, for example 6 hours, by using `-query-frontend.results-cache-ttl` command line option (or per tenant).
+This will guarantee that query results will use backfilled data at most after this period.  
```

Comment:
Language nits. I think that qualifying the alternative as possible is unnecessary, to me at least it makes it sound as if it's just a "possible" alternative and not a certain one.
```suggestion
To fix the cache results, the Mimir operator can manually flush the results cache.
An alternative is to decrease the time-to-live period for cache results from the default 7 days to a shorter period, for example 6 hours, by using `-query-frontend.results-cache-ttl` command line option (or per tenant).
This will guarantee that query results will use backfilled data at most after this period.
```

Reply:
```suggestion
To fix the cache results, the Mimir operator can manually flush the results cache.
An alternative is to decrease the time-to-live period for cache results from the default `7` days to a shorter period, for example 6 hours, by using the `-query-frontend.results-cache-ttl` command-line option (or per tenant).
This guarantees that query results will use backfilled data at most after this period.
```
What does "at most" mean here? Does it mean "mostly"?

Reply:
It means that you need to wait at most 6h after upload finished for uploaded data to appear in query results.